packages/sth/src/bin/hub.ts

```diff
@@ -59,7 +59,10 @@ const options: OptionValues & STHCommandOptions = program
     const resolveFile = (path: string) => path && resolve(process.cwd(), path);
 
     if (options.config) {
-        const configContents = await readConfigFile(options.config) as DeepPartial<STHConfiguration>;
+        const configFile = FileBuilder(options.config);
+
+        if (!(configFile.exists() && configFile.isReadable())) throw new Error("Unable to read config file");
+        const configContents = configFile.read() as DeepPartial<STHConfiguration>;
```

Comment:
can we add a config validation somewhere here? 
I have run STH with the negative port number and it tried to start the server:
![Screenshot from 2022-10-14 13-21-36](https://user-images.githubusercontent.com/8177865/195835413-8a15f9e6-1a10-4546-be3b-0103a201281d.png)

Reply:
I didn't touch STH yet :/